CameraTools/CameraTools.csproj

```diff
@@ -81,228 +81,57 @@
   <ItemGroup>
     <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\_LocalDev\KSPRefs\Assembly-CSharp.dll</HintPath>
-    </Reference>
-    <Reference Include="Assembly-CSharp-firstpass">
-      <HintPath>..\..\_LocalDev\KSPRefs\Assembly-CSharp-firstpass.dll</HintPath>
+      <HintPath>..\..\BDArmory\BDArmory\_LocalDev\KSPRefs\Assembly-CSharp.dll</HintPath>
```

Comment:
Yeah, there was a whole bunch of stuff included that was probably unnecessary. I suspect the original dev just included all the dlls in the KSPRefs.
The `HintPath` should be `..\..\_LocalDev\KSPRefs\...` though as the "usual" setup is
```
KSP_dev_folder
  |-- _LocalDev
  |-- BDArmory
  |-- CameraTools
```
where the `_LocalDev` folder contains links or copies of the various references and the required `.txt` files (depending on the OS), e.g.,
```
brett@br-workhorse:~/github/CameraTools/CameraTools$ ls -l ../../_LocalDev/
total 20
lrwxrwxrwx 1 brett brett 88 Oct 10  2020 BDARefs -> '/home/brett/.steam/steam/steamapps/common/Kerbal Space Program/GameData/BDArmory/Plugins'
lrwxrwxrwx 1 brett brett 92 Aug 14  2020 CameraToolsRef -> '/home/brett/.steam/steam/steamapps/common/Kerbal Space Program/GameData/CameraTools/Plugins/'
lrwxrwxrwx 1 brett brett 64 Apr 26 19:54 FARRefs -> /home/brett/Games/BAD-T/GameData/FerramAerospaceResearch/Plugins
lrwxrwxrwx 1 brett brett 79 Jun 21  2020 KSPRefs -> '/home/brett/.steam/steam/steamapps/common/Kerbal Space Program/KSP_Data/Managed'
-rw-rw-r-- 1 brett brett 63 Jun 21  2020 ksp_dir.txt
```

Reply:
Yeah, I have my localDev folder in the wrong place, will have to fix that.